apps/remix-ide/src/app/plugins/templates-selection/templates-selection-plugin.tsx

```diff
@@ -0,0 +1,250 @@
+    }
+    */
+    const createWorkspace = async (item) => {
+      const defaultName = await this.call('filePanel', 'getAvailableWorkspaceName', item.displayName)
+
+      const username = await this.call('settings', 'get', 'settings/github-user-name')
+      const email = await this.call('settings', 'get', 'settings/github-email')
+      const gitNotSet = !username || !email
+      let workspaceName = defaultName
+      let initGit = false
+      const modal: AppModal = {
+        id: 'TemplatesSelection',
+        title:  window._intl.formatMessage({ id: !isElectron() ? 'filePanel.workspace.create': 'filePanel.workspace.create.desktop' }),
+        message: await createModalMessage(defaultName, gitNotSet, (value) => workspaceName = value, (value) => initGit = !!value),
+        okLabel: window._intl.formatMessage({ id: !isElectron() ? 'filePanel.ok':'filePanel.selectFolder' }),
+      }
+      const modalResult = await this.call('notification', 'modal', modal)
+      if (!modalResult) return
+      this.emit('createWorkspaceReducerEvent', workspaceName, item.value, item.opts, false, (e, data) => {
+        if (e) {
+          const modal: AppModal = {
+            id: 'TemplatesSelection',
+            title:  window._intl.formatMessage({ id: !isElectron() ? 'filePanel.workspace.create': 'filePanel.workspace.create.desktop' }),
+            message: e.message,
+            okLabel: window._intl.formatMessage({ id: 'filePanel.ok' }),
+            cancelLabel: window._intl.formatMessage({ id: 'filePanel.cancel' })
+          }
+          this.call('notification', 'modal', modal)
+          console.error(e)
+        }
+
+      }, initGit)
+    }
+
+    const addToCurrentWorkspace = async (item) => {
+      this.emit('addTemplateToWorkspaceReducerEvent', item.value, item.opts, false, (e, data) => {
+        if (e) {
+          const modal: AppModal = {
+            id: 'TemplatesSelection',
+            title:  window._intl.formatMessage({ id: !isElectron() ? 'filePanel.workspace.create': 'filePanel.workspace.create.desktop' }),
+            message: e.message,
+            okLabel: window._intl.formatMessage({ id: 'filePanel.ok' }),
+            cancelLabel: window._intl.formatMessage({ id: 'filePanel.cancel' })
+          }
+          this.call('notification', 'modal', modal)
+          console.error(e)
+        }
+
+      })
+    }
+
+    return (
+      <RemixUIGridView
+        plugin={this}
+        styleList={""}
+        logo='/assets/img/bgRemi.webp'
+        enableFilter={true}
+        showUntagged={true}
+        showPin={false}
+        tagList={[
+          ['Solidity', 'danger'],
+          ['ZKP', 'warning'],
+          ['ERC20', 'success'],
+          ['ERC721', 'secondary'],
+          ['ERC1155', 'primary'],
+        ]}
+        title='Template explorer'
+        description="Select the template to create a workspace"
```

Comment:
it should also say `"Select the template to create a workspace or to add in current workspace"`